pkg/web/templates/partials/secure_envelope/secure_envelope_detail.html

```diff
@@ -1 +1,9 @@
-<div></div>
\ No newline at end of file
+{{ if .SecureEnvelope.Payload }}
```

Comment:
This is a basic implementation for now. The partial will be amended to style the payload and error in a follow on story.